napari/layers/image/_image_constants.py

```diff
@@ -28,6 +28,20 @@ class Interpolation(StringEnum):
     SPLINE16 = auto()
     SPLINE36 = auto()
 
+    @property
+    def view_subset(self):
+        return (
+            self.BICUBIC,
+            self.BILINEAR,
+            self.KAISER,
+            self.NEAREST,
+            self.SPLINE36,
+        )
+
+    @property
+    def in_view_subset(self):
+        return self in self.view_subset
+
```

Comment:
```suggestion
    @classmethod
    def view_subset(cls):
        return (
            cls.BICUBIC,
            cls.BILINEAR,
            cls.KAISER,
            cls.NEAREST,
            cls.SPLINE36,
        )
```

it's not quite as useful to access this subset from individual `Interpolation` instances... so maybe lets make it a class method (called as `Interpolation.view_subset()`